### ISSUE TITLE
strip propDecorators

### ISSUE THREAD

**Issue by IgorMinar** [alxhub/ngo#2]
currently we don't strip propDecorators which are heavily used by material and @angular/common directives and components. It's safe to strip them all without any exceptions.

```
NgClass.propDecorators = {
    'klass': [{ type: __WEBPACK_IMPORTED_MODULE_0__angular_core__["J" /* Input */], args: ['class',] },],
    'ngClass': [{ type: __WEBPACK_IMPORTED_MODULE_0__angular_core__["J" /* Input */] },],
};
```

**Comment by alxhub** [comment]
>  It's safe to strip them all without any exceptions.

Are you sure this is true? `@ngrx/effects` uses an `@Effect()` property decorator which is read at runtime.

**Comment by IgorMinar** [comment]
I meant that it's safe to remove all of angular's prop decorators

**Comment by alxhub** [comment]
Implemented in 5dd9616.
